posts/2022/sparse-PFT-gridding.ipynb

```diff
@@ -0,0 +1,12558 @@
+{
```

Comment:
<p>Here's what I would say about this dataset, let me know if this is not what you're looking for:</p><p> </p><p>This dataset represents output from the Community Land Model (CLM), the land model component of CESM. The particular variable here is gross primary production (GPP), which is essentially photosynthesis or carbon uptake by plants. Because CLM represents different plant functional types (PFTs) within a single model grid cell (up to 78 different types including natural vegetation and crops), this dataset includes information on how GPP varies by PFT. This way we can examine photosynthesis and how it varies across different plant types. To visualize this output, we often need to remap the information to a latitude/longitude grid, preserving plant type.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/esds/pull/159/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>